README.md

```
@@ -1,44 +1,77 @@
+| **Booking.com** | Flug, Hotel, Erlebnisse |
+| **Airbnb** | Unterkünfte, Erlebnisse |
+| **GetYourGuide** | Erlebnisse / Touren |
+| **Traveloka** | Hotel, Flug, Erlebnisse, Mietwagen, … |
+
+Zusätzlich: **manuelle Buchungen** (Flug, Hotel, Fähre, Erlebnis, Sonstiges) ohne Portal-Sync.
+
+Reisen ist **nicht** mit diesen Anbietern verbunden — Login erfolgt direkt beim Portal in einer eingebetteten Web-Ansicht.
 
 ## Architektur
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)